src/NetAppFiles/NetAppFiles/ChangeLog.md

```diff
@@ -18,6 +18,11 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+
+## Version 0.1.2
+* Addition of ProtocolType and MountTarget to volume
```

Comment:
It would be better to specify you are adding these paramter to which cmdlet

Reply:
`MountTargets` and `ProtocolTypes` are properties, and `ProtocolType` and `MountTarget` are parameters -- you should probably say that specifically in the changelog, just to be clear.

Reply:
Agreed. Updated the change log.